website/docs/tutorials/08-interacting-with-starknet/01-invoke.md

```diff
@@ -0,0 +1,37 @@
+# Invoking contracts
+
+## Overview
+This cheatcode allows you to send an invoke transaction with `@external` function entrypoint.
+
+The basic inputs that you need for the commands are:
+- [Signing credentials](#signing)
+- Contract address
+- Function name
+- Inputs to the function
+- Fee specification (concrete amount, or auto-estimation)
+- Network you want to target (i.e. its name or gateway URL)
+
+For detailed API description, see [invoke command reference](../../cli-reference.md#invoke).
+
+## Signing
+Credentials for paying the fee are needed, which are: 
+- Account address (which pays the fee)
+- Private key for that account (from ArgentX, Braavos etc.) - in hex string (i.e. 0xa1c359ef) form. This can be provided with `PROTOSTAR_ACCOUNT_PRIVATE_KEY` env variable or with a file on local filesystem containing that key in plaintext, in which case you should use `--private-key-path`.
+
+Custom signing logic is made possible by using custom signers - see details [here](../06-deploying/01-cli.md/#2-using-a-custom-signer-class).
```

Comment:
This should be abstracted because all transactions need to be signed.

Reply:
I don't think we should abstract the docs

Reply:
Ok. This section can be extracted later once the number of pages grows.